src/test/java/me/andrz/jackson/YamlFactoryObjectMapperFactory.java

```diff
@@ -0,0 +1,15 @@
+package me.andrz.jackson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+/**
+ * @author slasch
+ * @since 09.11.2015.
+ */
+class YamlObjectMapperFactory implements ObjectMapperFactory {
```

Comment:
@slavaschmidt Mismatch between class name (`YamlObjectMapperFactory `) and file name (`YamlFactoryObjectMapperFactory`)?